functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configurationfile/LocalChangesNewTestsSelectionExecutionWithConfigFileFunctionalTest.java

```diff
@@ -56,11 +56,8 @@ public void should_only_execute_new_tests_related_to_single_local_change_using_f
                 .createConfigFile()
             .enable();
 
-        project
-            .applyAsCommits("Disable surefire and enable just failsafe plugin");
-
         final Collection<TestResult> expectedTestResults = project
-            .applyAsLocalChanges("Adds new unit test");
+            .applyAsLocalChanges("Skip surefire and enable just failsafe plugin with sample integration test");
```

Comment:
both this change and the `LocalChangesNewTestsSelectionExecutionFunctionalTest.java` one is not necessary - it did the same thing, you just bring another tag for it doin't the same in one commit. 
I would keep it as it was